packages/venia-ui/lib/components/CartPage/PriceSummary/giftCardSummary.ee.js

```diff
@@ -39,14 +39,14 @@ export default props => {
     const cards = getGiftCards(props.data);
 
     return cards.value ? (
-        <>
+        <Fragment>
```

Comment:
Is this preferred over `<>`?

Reply:
Yes, it's a bit more explicit and portable. Generally, you can paste our code into a sandbox and it'll work. This particular transform isn't as popular as you might think.

Also, we've used `Fragment` almost everywhere else.